### PR TITLE
samples: net: wifi: support frdm_rw612 board

### PR DESCRIPTION
**Pull request by MaochenWang1**
add conf file for frdm_rw612 board.

### CODE REVIEW

File: samples/net/wifi/boards/frdm_rw612.conf [hunk 1]
```diff
@@ -0,0 +1,159 @@
+CONFIG_WIFI_NXP=y
```
Comment (jukkar) [inline samples/net/wifi/boards/frdm_rw612.conf:1]: Only put here configs that are related to this board. Most of the settings are related to application activity etc. non board related things.

**Review by jukkar — CHANGES_REQUESTED** [review]

File: samples/net/wifi/boards/frdm_rw612.conf (hunk 1 above)
Comment (MaochenWang1) [reply samples/net/wifi/boards/frdm_rw612.conf:1]: it's almost same as samples/net/wifi/boards/rd_rw612_bga.conf
and if these configs can't be put in this file, then where should put them?

Comment (jukkar) [reply samples/net/wifi/boards/frdm_rw612.conf:1]: Then we should remove extra settings from that file too.
These board files should only enable board specific settings if there are any.
It is in your application (in this sample it is the prj.conf file) that should contain generic settings.

Comment (MaochenWang1) [reply samples/net/wifi/boards/frdm_rw612.conf:1]: where should put the non-generic settings? I mean the application activity configs specific to this board, where should I put them?

Comment (jukkar) [reply samples/net/wifi/boards/frdm_rw612.conf:1]: If your generic settings are useful, then those should go to sample prj.conf file.
If you have for example DPP specific feature changes, then those could go for example to `overlay-dpp.conf`.
The reason why board specific files should contain only board specific settings, it that the sample should work similar way with default config regardless of the used board. It is annoying to get bug reports for things that are only enabled in one board.

Comment (MaochenWang1) [reply samples/net/wifi/boards/frdm_rw612.conf:1]: Hi @jukkar, different vendors have different features, and different boards have different size of SRAM and flash size. I mean for example, different board may want to use different overlay-optimize.conf.

Comment (MaochenWang1) [reply samples/net/wifi/boards/frdm_rw612.conf:1]: if we need to add "overlay-dpp.conf", then we still have to add "overlay-hostapd.conf", “overlay-wps.conf", "overlay-mbedtls.conf", "overlay-optimize.conf", "overlay-power-management.conf". then the build cmd is complicated, can we have some way to use one overlay.conf to build? 
and then each conf will be generic for different boards, if one developer change one conf for its own board, may have side effect on other boards.

**Pull request closed without merging**